source/renderer/app/components/loading/syncing-connecting/SyncingConnectingTitle.scss

```diff
@@ -1,32 +1,32 @@
-@import '../../../themes/mixins/animations';
-
 .component {
-  bottom: 64px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  left: 0;
   position: absolute;
-  text-align: center;
+  top: calc(50% + 67.5px);
   width: 100%;
-}
-
-.syncing {
-  color: var(--theme-syncing-text-color);
-}
-
-.headline {
-  font-family: var(--font-regular);
-  font-size: 18px;
-}
 
-.connecting {
-  color: var(--theme-connecting-text-color);
-  .headline {
-    overflow: visible;
-    &:not(.withoutAnimation) {
-      @include animated-ellipsis($width: 20px);
-    }
+  .textContent {
+    font-family: var(--font-regular);
+    font-size: 20px;
+    font-weight: 600;
+    letter-spacing: 1px;
+    line-height: 1.2;
+    text-transform: uppercase;
   }
-}
 
-.component,
-.headline {
-  -webkit-backface-visibility: hidden;
+  .title {
+    color: var(--theme-syncing-connecting-title-color);
+    margin-bottom: 5px;
+    margin-top: 30px;
+  }
+  .subTitle1 {
+    color: var(--theme-syncing-connecting-subTitle1-color);
+    margin-bottom: 5px;
+  }
+  .subTitle2 {
```

Comment:
Shouldn't this color be `#eb2256` instead?

![image](https://user-images.githubusercontent.com/1504716/68668220-764ee700-0526-11ea-9299-da3c7a99254d.png)